viewer-prototype/.eslintrc.js

```diff
@@ -0,0 +1,37 @@
+/** @type {import('eslint').Linter.Config} */
```

Comment:
ATM I think only the `viewer-prototype` folder is covered, since there is no `.eslintrc.js` in the repo root. Probably not too important since there is little JS/TS code elsewhere in the repo?

Reply:
Indeed, I only implemented it in the viewer-prototype for now. There is nothing much elsewhere, so we'll add it as we need it I think.